src/CalibrateCMP/KiDUtils.jl

```diff
@@ -288,6 +297,15 @@ function create_kid_parameters(
         w1 = w1,
         t1 = t1,
         p0 = p0,
+        z_0 = z_0,
+        z_1 = z_1,
+        z_2 = z_2,
+        rv_0 = rv_0,
+        rv_1 = rv_1,
+        rv_2 = rv_2,
+        tht_0 = tht_0,
+        tht_1 = tht_1,
+        tht_2 = tht_2,
```

Comment:
```suggestion
        w1,
        t1,
        p0,
        z_0,
        z_1,
        z_2,
        rv_0,
        rv_1,
        rv_2,
        tht_0,
        tht_1,
        tht_2,
```

Since the variables have the same name as the keyword args and come after the semicolon, you should be able to pass them in without the `=`.